test/index.test.js

```diff
@@ -49,32 +51,46 @@ tape('prepare queries', function (assert) {
     assert.end();
 });
 
-tape('mocking [ELB]', function (assert) {
-    AWS.mock('CloudWatch', 'getMetricStatistics', function (params, callback) {
+tape('mocking [CloudWatch]', function (assert) {
 
+    AWS.CloudWatch = MockCloudWatch;
+    function MockCloudWatch() {}
+
+    MockCloudWatch.prototype.getMetricStatistics = function (params, callback) {
         metricdatapoint.forEach(function (i) {
             if (params.MetricName === i.Label) callback(null, i);
         });
-    });
+    };
     assert.end();
 });
+
 tape('ELB metrics', function (assert) {
     var obj = {
-        startTime: 1471692377978,
-        endTime: 1471698014705,
+        startTime: 1473838245009,
+        endTime: 1473838232083,
         region: 'us-east-1',
         elbname: 'abc'
     };
     var datapoints = prepareQueries(obj);
+    var expectedPecentages = {period: '12s',
+    requestPerSecond: '4/s',
+    percent2xx: '50 %',
+    percent3xx: '25 %',
+    percent4xx: '22.92 %',
+    percent5xx: '2.08 %',
+    avgLatency: 0.4};
```

Comment:
This is minor, but for avgLatency -- it might be better to demonstrate average latency in a test with a variety of different latency datapoints. Say, if your datapoints for latency are `1, 5, 6` the average would be 4.

In [your fixture](https://github.com/mapbox/elb-metrics/blob/8ed5fb2927c9dd394166d5b4f1bbd3e5f1bc7075/test/fixture/datapoints.json#L100-L114), all of the latency datapoints are the same thing (`0.4`). Average latency ending up at `0.4` doesn't really represent that your average function is working... because, the tests would pass if you just selected a datapoint at random, instead of averaging them all. 

Again, this is minor, but I prefer to write test cases that are very explicit in this way.